README.md

```diff
@@ -17,13 +17,13 @@ Refer to [deployment instructions here](./doc/deploy/deployment.md) on how to de
 
 ## Release Process
 
-1. Update version in [CONTROLLER_VERSION file](https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/main/CONTROLLER_VERSION) to the new release version.
+1. Run the [mcad-release.yml](https://github.com/project-codeflare/actions/workflows/mcad-release.yml) action.
```

Comment:
I think we need explicit steps / screen shots on how to do that.  This statement assumes that one might know how to do it. Also, if there are any permissions / privileges that are required to do so, they need to be documented, and also the process to request them.

Reply:
Sure will update the relevant steps and info to trigger the Workflow